### DISCUSSION TITLE
Integrate JBang with https://mvnrepository.com/

### DISCUSSION THREAD

**Discussion by linux-china** [jbangdev/jbang#1165] [Ideas]
Most developers use https://mvnrepository.com/ to search Maven artifacts.  Even search from Google, for example `maven commons-lang3`, and will be redirected to mvnrepository.com. 

The Maven artifact detail page, for example  https://mvnrepository.com/artifact/org.apache.commons/commons-lang3/3.12.0  Screenshot as following: 

![image](https://user-images.githubusercontent.com/46711/147502310-0fb87a94-4a05-4705-a873-707f1516f480.png)

Is it impossible to talk with mvnrepository.com author and add JBang tab and with following directive? 

```
//DEPS org.apache.commons:commons-lang3:3.12.0
```

**Comment by quintesse** [comment]
That would definitely be great! :+1:

**Reply by maxandersen** [reply]
Yeah. I opened https://github.com/sonatype-nexus-community/search-maven-org/pull/137 for search.maven.org. Pr approved but still waiting. 

If we can find similar for mvnsearch it would be nice.

**Comment by maxandersen** [comment]
sent a mail to their info email - just to realize I 23 months ago asked to get the short gradle format added - never got a reply but they did add the short format so there is hope :)

**Comment by linux-china** [comment]
Just find  [Maven Central Search](https://github.com/mthmulders/mcs)  CLI, and it's friendly to JBang.

`mcs search commons-lang3 | sed '6,$s/^  /\/\/DEPS /'` result: 


![image](https://user-images.githubusercontent.com/46711/147607570-ff677ae7-72b5-4493-a765-0eb32288750d.png)


```
function jmcs() {
    /usr/local/opt/mcs/bin/mcs search "$@" | sed '6,$s/^  /\/\/DEPS /'
}
```

**Reply by maxandersen** [reply]
Try 'jbang gavsearch@jbangdev' :)
